kohese-portal/package.json

```diff
@@ -24,103 +24,102 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^6.1.0",
-    "@angular/cdk": "^6.4.7",
-    "@angular/common": "^6.1.0",
-    "@angular/compiler": "^6.1.0",
-    "@angular/core": "^6.1.0",
-    "@angular/flex-layout": "^6.0.0-beta.18",
-    "@angular/forms": "^6.1.0",
-    "@angular/http": "^6.1.0",
-    "@angular/material": "^6.4.7",
-    "@angular/platform-browser": "^6.1.0",
-    "@angular/platform-browser-dynamic": "^6.1.0",
-    "@angular/pwa": "^0.8.4",
-    "@angular/router": "^6.1.0",
-    "@angular/service-worker": "^6.1.9",
-    "@auth0/angular-jwt": "^2.0.0",
-    "@swimlane/ngx-datatable": "^13.1.0",
-    "@tinymce/tinymce-angular": "^3.2.0",
-    "@types/socket.io-client": "^1.4.32",
-    "angular-split": "^1.0.0-rc.3",
-    "angular-tree-component": "^8.0.0",
+    "@angular/animations": "^9.1.13",
+    "@angular/cdk": "^9.2.4",
+    "@angular/common": "^9.1.13",
+    "@angular/compiler": "^9.1.13",
+    "@angular/core": "^9.1.13",
+    "@angular/forms": "^9.1.13",
+    "@angular/localize": "^9.1.13",
+    "@angular/material": "^9.2.4",
+    "@angular/platform-browser": "^9.1.13",
+    "@angular/platform-browser-dynamic": "^9.1.13",
+    "@angular/router": "^9.1.13",
+    "@angular/service-worker": "^9.1.13",
+    "@auth0/angular-jwt": "^4.2.0",
+    "@schematics/angular": "^9.1.12",
+    "@tinymce/tinymce-angular": "^3.6.1",
+    "@types/socket.io-client": "^1.4.36",
+    "angular-split": "^3.0.3",
+    "angular-tree-component": "^8.5.6",
     "angular2-virtual-scroll": "^0.4.16",
     "bcryptjs": "^2.4.3",
-    "bootstrap": "^4.1.3",
+    "bootstrap": "^4.6.0",
+    "chokidar": "^3.5.2",
     "commonmark": "^0.28.1",
-    "core-js": "^2.5.4",
-    "d3": "^5.7.0",
-    "express-jwt": "^5.3.1",
+    "core-js": "^2.6.12",
+    "d3": "^5.16.0",
+    "express-jwt": "^5.3.3",
     "font-awesome": "^4.7.0",
     "jasmine-node": "^3.0.0",
-    "jquery": "^3.3.1",
-    "jsonwebtoken": "^8.3.0",
-    "jssha": "^2.3.1",
-    "karma-coverage-istanbul-reporter": "^1.4.1",
-    "level": "^4.0.0",
-    "level-js": "^3.0.0",
-    "levelup": "^3.1.1",
+    "jquery": "^3.6.0",
+    "jsonwebtoken": "^8.5.1",
+    "jssha": "^2.4.2",
+    "level": "^7.0.0",
+    "level-js": "^6.0.0",
+    "leveldown": "^6.0.0",
+    "levelup": "^5.0.1",
     "mark.js": "^8.11.1",
-    "morgan": "^1.7.0",
+    "morgan": "^1.10.0",
     "ng2-dragula": "^1.5.0",
-    "ng2-file-upload": "^1.3.0",
+    "ng2-file-upload": "^1.4.0",
     "ng2-tree": "^2.0.0-rc.11",
-    "ngx-infinite-scroll": "^6.0.1",
-    "ngx-markdown": "^6.2.1",
-    "ngx-toastr": "^9.1.0",
-    "node-fetch": "^2.6.0",
+    "ngx-infinite-scroll": "^6.9.1",
```

Comment:
Scrolling area that uses infinite scroll results in a lot of text in the console.

Reply:
Documented as Issue: Updated Infinite Scroll prints a lot of text to console (e2fcc4b0-044a-11ec-8a16-d5093e66c866)